index.html

```diff
@@ -58,15 +66,18 @@ <h1 class="project-title">{{title}}</h1>
         <ul>
           <li class="tab" data-content="projects"><a href="#" class="icon-home"> Home</a></li>
           <li class="tab" data-content="testimonials"><a href="#" class="icon-pencil"> Testimonials</a></li>
+          <li class="tab" data-content="admin"><a href="#" class="icon-spinner7"> Admin</a></li>
           <li><a href="http://twitter.com/Kujakuseki" target="_blank" class="icon-twitter"> Twitter</a></li>
           <li><a href="https://github.com/broxsonl" target="_blank" class="icon-github"> GitHub</a></li>
+          <!--TODO: Add a Linkedin option to navbar-->
         </ul>
       </nav>
     </header>
     <!-- <div class="clearfix"></div> -->
     <main class="clearfix">
       <!--PROJECTS SECTION -->
       <section id="projects" class="tab-content">
+        <!--TODO: Remove <br> tags and correct the spacing issues with CSS-->
```

Comment:
I bet margins and padding would help solve this problem, or if that doesn't fit the bill- relative/ absolute positioning